modules/greenbidsRtdProvider.js

```diff
@@ -85,14 +103,24 @@ function getFalseBidders(bidders) {
     .map(([bidder]) => bidder);
 }
 
-function createPayload(reqBidsConfigObj) {
+function stripAdUnits(adUnits) {
+  const stripedAdUnits = deepClone(adUnits);
+  return stripedAdUnits.map(adUnit => {
+    adUnit.bids = adUnit.bids.map(bid => {
+      return { bidder: bid.bidder };
+    });
+    return adUnit;
+  });
+}
```

Comment:
Looks like this gets sent to your server, and you just only want bidder name.

Obviously I have no idea what your server is doing, but you could just return bids as array of strings instead of array of objects with one `bidder` key. Save some space.

Also, there may be quite a lot of other stuff on the adUnit, do you want all of that in your strippedAdUnits also? Like ORTB2Imp stuff, mediaTypes, etc?